minify.js

```diff
@@ -260,7 +260,7 @@ module.exports = function express_minify(options) {
                 res.setHeader('Content-Type', 'text/css');
             } else if (coffee_match.test(contentType)) {
                 type = TYPE_COFFEE;
-                res.setHeader('Content-Type', 'text/javascript');
+                res.setHeader('Content-Type', 'application/javascript');
```

Comment:
@SummerWish,

This knocks out other browser compatibility... correct mime type is `text/javascript` if it is not to be run e.g. just parsed or distributed... all browsers are capable of supporting this however the application mime type is a newer standard for execution to be handle by the browser in the browser.